opentelemetry-otlp/CHANGELOG.md

```diff
@@ -2,6 +2,8 @@
 
 ## vNext
 
+- Make `force_flush()` in `PushMetricExporter` synchronous
```

Comment:
This should go in the changelog of the opentelemetry_sdk crate.

Reply:
moved and marked as breaking change